log_drains/add.go

```diff
@@ -7,17 +7,48 @@ import (
 	"gopkg.in/errgo.v1"
 )
 
-func Add(app string, params scalingo.LogDrainAddParams) error {
+type AddAddonOpts struct {
+	WithAddons bool
+	AddonID    string
+}
+
+func Add(app string, opts AddAddonOpts, params scalingo.LogDrainAddParams) error {
```

Comment:
You didn't addressed my comment here.  What I meant is that I would prefer you keep 2 arguments for this method: `app string` and the optional params. And these params being in a struct. It's semantically more correct. Currently you have a `AddAddonOpts` params which makes no sens when you just want to add drains to an application. 

For instance you could use embedded struct. Here is an example of what embedded structs are: https://play.golang.org/p/wszdms3OpBU
You create a new srtucture which embeds `scalingo.LogDrainAddParams` and add the fields from `AddAddonOpts`